tests/system/HTTP/ResponseCookieTest.php

```diff
@@ -314,6 +316,30 @@ public function testCookieInvalidSameSite()
         ]);
     }
 
+    public function testSetCookieConfigCookieIsUsed()
+    {
+        /** @var \Config\Cookie $config */
```

Comment:
there is alias for it, so use `CookieConfig` for shorter:

```suggestion
        /** @var CookieConfig $config */
```

Reply:
Done.